src/theme/ItaliaTheme/Blocks/_highlitedContent.scss

```diff
@@ -17,6 +17,8 @@
 
   .card-title {
     a {
+      display: inline-block; //per avere il bordo tutto unito al focus di un link, ad esempio nelle card
+      color: $link-color;
```

Comment:
Chiedo per scrupolo se hai testato: settando il colore qua funziona ancora la variazione di colore all'hover? Ho avuto esperienze in passato in cui settando il colore senza settare anche l'hover finiva che prendeva sempre quello settato